aquarius/ddo_checker/shacl_schemas/v4/remote_4.0.0.ttl

```diff
@@ -0,0 +1,351 @@
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <http://schema.org/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+schema:DDOShape
+    sh:targetClass schema:DDO ;
+    sh:property [
+        sh:path schema:id ;
+        sh:datatype xsd:string ;
+        sh:pattern "^did\\:op\\:(.*)$" ;
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
+    ] ;
+    sh:property [
+        sh:path schema:version ;
+        sh:datatype xsd:string ;
+        sh:pattern "^(.*)$" ;
```

Comment:
Should we reject non-SemVer versions?

Reply:
yes, I think in a previous version we had the right pattern. I'll put that one back.

Reply:
actually, I remembered that we only use our own files. That means the version first validated by having that file in our data, e.g. "remote_4.0.0.ttl". So the pattern is not really relevant, since we hold a list of all the allowed versions.